my_list.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
+
+  attr_reader :list
+
+  def each
+    puts yield @list if block_given?
+  end
+end
```

Comment:
- [ ] The #each method should yield successive members of @list, not individual ones, kindly consider modifying your code so `irb> list = MyList.new(1, 2, 3, 4)` returned `#<MyList: @list=[1, 2, 3, 4]>`